CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ to the [Haskell Package Versioning Policy][pvp].
 
 ## [Unreleased]
 
+## [1.0.1.0] - 2022-02-09
```

Comment:
The version number is surrounded in square brackets because it's a Markdown link. You need to make this change at the bottom of this file:

```diff
-[Unreleased]: [https://github.com/awakesecurity/nix-graph/compare/v1.0.0.0...HEAD]
+[Unreleased]: [https://github.com/awakesecurity/nix-graph/compare/v1.0.1.0...HEAD]
+[1.0.1.0]: https://github.com/awakesecurity/nix-graph/releases/tag/v1.0.1.0
 [1.0.0.0]: https://github.com/awakesecurity/nix-graph/releases/tag/v1.0.0.0
 [changelog]: https://keepachangelog.com/en/1.0.0/
 [pvp]: https://pvp.haskell.org/
```

Reply:
oh, right! duh!